flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/codegen/GenerateUtils.scala

```diff
@@ -572,12 +572,15 @@ object GenerateUtils {
     * @param ctx code generator context which maintains various code statements.
     * @param fieldType type of field
     * @param fieldTerm expression term of field to be unboxed
+    * @param unboxingTerm unboxing/conversion term
     * @return internal unboxed field representation
     */
   def generateInputFieldUnboxing(
       ctx: CodeGeneratorContext,
       fieldType: LogicalType,
-      fieldTerm: String): GeneratedExpression = {
+      fieldTerm: String,
```

Comment:
`fieldTerm` -> `inputTerm`?

Reply:
I think the method should be refactored in the future. In Flink planner, this method was intended to perform `genToInternalIfNeeded`. Now the concepts are mixed up.